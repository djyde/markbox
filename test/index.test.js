const markbox = require('../lib/markbox')

describe('Markbox', () => {


  const content = `

Code from markdown:

\`\`\`js
document.write('Code from markdown!')
\`\`\`

`

  test('parse', async () => {
    jest.setTimeout(15000)
    const html = await markbox.parse(content)
    expect(html).toMatchSnapshot()
  })
})