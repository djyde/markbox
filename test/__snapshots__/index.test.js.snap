// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Markbox parse 1`] = `
"<p>Code from markdown:</p>
<iframe class=\\"codesandbox\\" sandbox=\\"allow-modals allow-forms allow-popups allow-scripts allow-same-origin\\" src=\\"https://codesandbox.io/embed/93ow5vp8o?\\"></iframe>"
`;
